lib/socketlabs/injectionapi/send_result.rb

```diff
@@ -308,6 +308,14 @@ def self.enum
             :value =>37,
             :message =>"SDK Validation Error: Expected messageType of basic or bulk"
 
+          },
+
+        # SDK Validation Error: Message contains invalid metadata
+        "MessageValidationInvalidMetadata" =>
+          {
+            :name =>"MessageValidationInvalidMetadata",
+            :value =>38,
```

Comment:
This value increments based on the last value in the hash. Felt worth calling out as I'm unsure of its significance.